acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,109 @@
+@parallel=false
+# for https://folio-org.atlassian.net/browse/MODORDSTOR-402
```

Comment:
url and annotation is not need in here. Just keep ticket number

Reply:
updated

Reply:
probably forget to push comments, I don't see changes